File: 03-conditional-statements-advanced/exercises-submitted-evaluated/11.fruitShop.js
function fruitShop(input) {
    let fruit = input[0];
    let day = input[1];
    let quantity = Number(input[2]);
    let total = 0;
    let banana = 0;
    let apple = 0;
    let orange = 0;
    let grapefruit = 0;
    let kiwi = 0;
    let pineapple = 0;
    let grapes = 0;
    switch (day) {
        case ("Monday"):
        case ("Tuesday"):
        case ("Wednesday"):
        case ("Thursday"):
        case ("Friday"):
            banana = 2.5;
            apple = 1.2;
            orange = 0.85;
            grapefruit = 1.45;
            kiwi = 2.7;
            pineapple = 5.5;
            grapes = 3.85;
            switch (fruit) {
                case ("banana"):
                    total = banana * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("apple"):
                    total = apple * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("orange"):
                    total = orange * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("grapefruit"):
                    total = grapefruit * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("kiwi"):
                    total = kiwi * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("pineapple"):
                    total = pineapple * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("grapes"):
                    total = grapes * quantity;
                    console.log(total.toFixed(2));
                    break;
                default:
                    console.log("error");
                    break;
            }
            break;
        case ("Saturday"):
        case ("Sunday"):
            banana = 2.7;
            apple = 1.25;
            orange = 0.9;
            grapefruit = 1.6;
            kiwi = 3;
            pineapple = 5.6;
            grapes = 4.2;
            switch (fruit) {
                case ("banana"):
                    total = banana * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("apple"):
                    total = apple * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("orange"):
                    total = orange * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("grapefruit"):
                    total = grapefruit * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("kiwi"):
                    total = kiwi * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("pineapple"):
                    total = pineapple * quantity;
                    console.log(total.toFixed(2));
                    break;
                case ("grapes"):
                    total = grapes * quantity;
                    console.log(total.toFixed(2));
                    break;
                default:
                    console.log("error");
                    break;
            }
            break;
        default:
            console.log("error");
            break;
    }

}
fruitShop(["apple", "Tuesday", "2"]);
fruitShop(["orange", "Sunday", "3"]);
fruitShop(["kiwi", "Monday", "2.5"]);
fruitShop(["grapes", "Saturday", "0.5"]);
fruitShop(["tomato", "Monday", "0.5"]);

File: softuni-python-basics/03_conditional_statements_advanced/11_fruit_shop.py
product = input()
day = input()
quantity = float(input())
price = 0
valid = True

if day == "Monday" or day == "Tuesday" or day == "Wednesday" or day == "Thursday" or day == "Friday":
    if product == "banana":
        price = 2.5
    elif product == "apple":
        price = 1.2
    elif product == "orange":
        price = 0.85
    elif product == "grapefruit":
        price = 1.45
    elif product == "kiwi":
        price = 2.7
    elif product == "pineapple":
        price = 5.5
    elif product == "grapes":
        price = 3.85
    else:
        valid = False
elif day == "Saturday" or day == "Sunday":
    if product == "banana":
        price = 2.7
    elif product == "apple":
        price = 1.25
    elif product == "orange":
        price = 0.9
    elif product == "grapefruit":
        price = 1.6
    elif product == "kiwi":
        price = 3
    elif product == "pineapple":
        price = 5.6
    elif product == "grapes":
        price = 4.2
    else:
        valid = False
else:
    valid = False

if not valid:
    print("error")
else:
    total = price * quantity
    print(f"{total:.2f}")
